g2p/convert.py

```diff
@@ -39,15 +42,15 @@ def run(self):
 
```

Comment:
If one passes a file that does not adhere to the desired xml structure, `tree.findall(".//lemma")` would be an empty array and the resulting output file is an empty file. I would add an assertion on the length before starting this loop.

Reply:
Added  this check in the last commit